src/Common/src/CoreLib/System/ReadOnlySpan.Fast.cs

```diff
@@ -23,7 +23,7 @@ namespace System
     /// or native memory, or to memory allocated on the stack. It is type- and memory-safe.
     /// </summary>
     [DebuggerTypeProxy(typeof(SpanDebugView<>))]
-    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    [DebuggerDisplay("{ToString(),raw}")]
```

Comment:
By default I thought VS rendered most things with ToString().  I could be wrong, but what happens if you remove the DebuggerDisplay attribute entirely?

Reply:
![image](https://user-images.githubusercontent.com/1697911/36506119-7f9bbbfa-170a-11e8-93fd-eb0f6192ad60.png)
This happens (special characters are not escaped)

Reply:
If we don't have it, we end up with the issue mentioned here: https://github.com/dotnet/corefx/pull/26726#issuecomment-362166770

![image](https://user-images.githubusercontent.com/6527137/36506444-848baebc-170b-11e8-80ae-658469125067.png)

Reply:
Ah. Thanks.